tests/test_subannual.R

```diff
@@ -0,0 +1,293 @@
+library(testthat)
+library(tsbox)
+
+t3 <- ts_rbind(ts_dt(mdeaths), AirPassengers)   ##### this function contains error, should have both
```

Comment:
```r
##### this function contains error, should have both
```
1. Can you open an issue, rather than commenting in the code this file should be clean.
2. Comment: Use one `#` only, let's follow http://style.tidyverse.org/. You could also run the new `styler` package (https://cran.r-project.org/web/packages/styler/index.html) to do it for you.

Reply:
To open an issue, just go to `issues` on `github.com/christophsax/tsbox` and select `new issue`

Reply:
Thanks, I will open an issue on this.